pkg/util/sys/device.go

```diff
@@ -43,6 +43,14 @@ type Partition struct {
 	Filesystem string
 }
 
+type CephVolumeDeviceData struct {
+	DevicePath string `json:"device path"`
+	Size       string `json:"size"`
```

Comment:
It looks like size, rotational, and model are also on the LocalDisk type. So the only new properties c-v is giving us is device path and available?

Reply:
yes, even "devicepath" can be inferred from the "name" property in the LocalDisk struct.
The only really "new" property is "available" which the result of several checks executed inside c-v.
My first aim was to include just the "available" field making the same calculations that are done in c-v. I commented this in the orchestrators meeting we had a couple of weeks ago (Sept 30), and we decided to include in the configmap the information coming from c-v.
This is to have only one source for the information needed to deal with storage devices in Ceph, and this source is c-v